### ISSUE TITLE
[Bug]: Infrastructure diagram issues

### ISSUE THREAD

**Issue by jstiops** [AsBuiltReport/AsBuiltReport.Veeam.VBR#186]
### Veeam Backup & Replication version

12.2

### Bug description

I'm using module version v0.8.11 and latest of diagrammer modules and other dependencies.
Generating the .html report on Windows Server 2022.

two things:
First: 
The diagram is begin saved in the PSScritpRoot, not in the specified output folder.
the .png infrastructure diagram looks like the screenshot. See the weird line drawn on the right.

Object Storage is not used in my case, but it's shown on the diagram. Not that much of an issue, but if possible, would be nice to just not show in that case.

### Command-line input

New-AsBuiltReport -Report Veeam.VBR -Target vbrserver -Format Html -OutputFolderPath 'C:\Utils\AsBuiltReport-Veeam\output' -ReportConfigFilePath 'C:\Utils\AsBuiltReport-Veeam\VeeamAsBuiltconfig.json' -Verbose -timestamp

### Steps to reproduce

ran the command with latest versions as of today.

### Expected behaviour

infrastructure diagram placed in output folder specified along with reports.
Issue 2 to be addressed

### Screenshots

![image](https://github.com/user-attachments/assets/a23d4092-5337-4c59-9101-aed9902769fb)

### Operating System

Windows Server 2022

### PowerShell Version

Name                           Value
----                           -----
PSVersion                      5.1.20348.2400
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.20348.2400
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

### PowerShell Modules

Name                    Version
----                    -------
AsBuiltReport.Core      1.4.0
AsBuiltReport.Veeam.VBR 0.8.11
Diagrammer.Core         0.2.11
PScribo                 0.10.0
Veeam.Diagrammer        0.6.12

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://www.asbuiltreport.com/about/contributing/#reporting-issues-and-bugs).

**Comment by rebelinux** [comment]
Could you run these cmdlet and send me the logs?

```powershell
update-Module Diagrammer.core -force    

update-Module Veeam.Diagrammer -force

Import-Module Diagrammer.Core -Force 

Import-Module Veeam.Diagrammer -Force
```

 Enable powershell debugging:

```powershell
$global:VerbosePreference = 'Continue'
$global:DebugPreference = 'Continue'
Start-Transcript -Path .\Output.log
New-VeeamDiagram -Target backupserver -Username "Username" -Password Password -Format png -Direction top-to-bottom -OutputFolderPath OutputFolder -EdgeType polyline -DiagramType Backup-Infrastructure  <-- change this part with your parameters-->
Stop-Transcript
```

The commands generate the Output.log file which you can send to me at:

jcolonf@zenprsolutions.com

After saving the log, reset powershell to default settings:

```powershell
$global:VerbosePreference = 'SilentlyContinue'
$global:DebugPreference = 'SilentlyContinue'
```

**Comment by rebelinux** [comment]
Any news about this?

**Comment by jstiops** [comment]
sorry for the delay. I've sent you the requested information.

**Comment by rebelinux** [comment]
Fixed in version 0.6.17

https://github.com/rebelinux/Veeam.Diagrammer/blob/dev/CHANGELOG.md#0617---2024-11-28

**Comment by rebelinux** [comment]
Latest release fix this:

The diagram is begin saved in the PSScritpRoot, not in the specified output folder.

Please test version 0.8.13 and let me know!
